packages/shared_preferences/shared_preferences_windows/test/shared_preferences_windows_test.dart

```diff
@@ -87,23 +88,23 @@ void main() {
 /// path it returns is a root path that does not actually exist on Windows.
 class FakePathProviderWindows extends PathProviderPlatform
     implements PathProviderWindows {
-  VersionInfoQuerier versionInfoQuerier;
+  late VersionInfoQuerier versionInfoQuerier;
 
   @override
   Future<String> getApplicationSupportPath() async => r'C:\appsupport';
```

Comment:
This is nullable in the interface, so it would be clearer to declare it that way for the override.

Reply:
you mean `getApplicationSupportPath()` ?